pkg/operator/apis/monitoring/v1/types.go

```diff
@@ -1019,6 +1041,14 @@ type ClusterPodMonitoringSpec struct {
 	TargetLabels TargetLabels `json:"targetLabels,omitempty"`
 	// Limits to apply at scrape time.
 	Limits *ScrapeLimits `json:"limits,omitempty"`
+	// FilterRunning will drop any pods that are in the "Failed" or "Succeeded"
+	// pod lifecycle.
+	// See: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase
+	// This prevents noisy errors when attempting to scrape Suceeded pods from
```

Comment:
It's not errors really, the succedded (completed) jobs which are not cleaned up report normal metrics just perhaps not relevant in majority of cases (if not all). (:

Reply:
Done.